src/components/button/button.jsx

```diff
@@ -15,8 +16,22 @@ import styled from "styled-components";
  * @param onClick callback function to run when btn is clicked
  */
 
-// TODO integrate icons when all are catalogued
-// TODO update bg-opacity when theme color mapping done
+const StyledButton = styled.button`
+  border: none;
+  border-radius: 8px;
+  opacity: ${(props) => props.disabled && "0.5"};
+  width: ${(props) => props.size};
+  transition: opacity 0.2s;
+  &:hover {
+    opacity: ${(props) => !props.disabled && "0.8"};
+    cursor: ${(props) => props.disabled && "default"};
+  }
+`;
+
+const StyledText = styled.h3`
+  margin: 19px 0px;
+  font-weight: 700;
```

Comment:
there are buttons that need different margins.  Would it be possible to add a prop and then also change the default to 20px.  The bs spacings are in multiples of 4px and the buttons are next to components styled with bs paddings and margins with 20 px.

Here is a way to add a prop to the Styled Text (would also need to add margin to the Button props)
`
  Button.Text = styled.h3`
    margin: ${(props) =>
      props.margin ||
      "20px 0px"}; 
    font-weight: 700;
  `;
`
here is how one would implement it
`
   <Button
        size="lg"
        buttonKind="tertiary"
        margin="12px 0px"
        onClick={regenerateSeedPhrases}
      >
        Regenerate
      </Button>
`
and passing props here
`export const Button = ({
  size,
  buttonKind,
  disabled,
  dangerText,
  leftIcon,
  rightIcon,
  onClick,
  children,
  margin,
}) => { ....rest of component
`

Reply:
other than that....looks good to me:)

Reply:
Since everything else looks great, I'm going to go ahead and merge this branch so we all have access to the changes.  The edit requested above only affects a couple of things and can be added later if needed.